orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/LogoutService.java

```diff
@@ -80,21 +83,17 @@ public LogoutService(
 
     private APIGatewayProxyResponseEvent generateLogoutResponse(
             URI logoutUri,
-            Optional<String> state,
+            LogoutReason logoutReason,
             TxmaAuditUser auditUser,
             Optional<String> clientId,
             Optional<String> rpPairwiseId) {
-        LOG.info("Generating logout response using URI: {}", logoutUri);
-        var uri =
-                state.map(s -> buildURI(logoutUri, Map.of(STATE_PARAMETER_KEY, s)))
-                        .orElse(logoutUri);
-
-        sendAuditEvent(auditUser, clientId, rpPairwiseId);
+        LOG.info("Generating logout response using URI: {}", logoutUri.getPath());
```

Comment:
I think this needs host and path to be meaningful?

Reply:
Yes, fixed